packages/react-ui/components/TokenInput/TokenInput.tsx

```diff
@@ -180,8 +200,8 @@ export class TokenInput<T = string> extends React.PureComponent<TokenInputProps<
     if (prevProps.selectedItems.length !== this.props.selectedItems.length) {
       LayoutEvents.emit();
     }
-    if (!this.isCursorVisibleForState(prevState) && this.isCursorVisible && !this.isEditingMode) {
-      this.tryGetItems(this.state.inputValue);
+    if (!this.isCursorVisibleForState(prevState) && this.isCursorVisible) {
+      this.tryGetItems();
```

Comment:
```suggestion
      this.tryGetItems(this.isEditingMode ? '' : this.state.inputValue);
```

Чтобы не менять прошлого поведения, кажется, лучше сделать так.

Reply:
Поведение не изменилось, но можно и так сделать)

Reply:
Поменялся при фокусе на инпут, если в нем уже был какой-то текст. Раньше поиск стартовал с этим текстом, а до последней правки - по пустой строке.

Сейчас все норм.